sidebars.js

```diff
@@ -32,7 +32,8 @@ module.exports = {
       label: 'Sorting Algorithms',
       items: [
         'sorting-algorithms/sorting-algorithms',
-        'sorting-algorithms/bubble-sort'
+        'sorting-algorithms/bubble-sort',
+        '/sorting-algorithms/merge-sort'
```

Comment:
Your build is failing because of this. Don't keep relative paths. Just keep `sorting-algorithms/merge-sort` here